src/components/DayPickerNavigation.jsx

```diff
@@ -138,8 +147,9 @@ function DayPickerNavigation({
             ]),
           )}
           aria-label={phrases.jumpToPrevMonth}
-          onClick={onPrevMonthClick}
+          onClick={hasPrev ? onPrevMonthClick : () => {}}
```

Comment:
let's have this be undefined if it's disabled, so something like `hasPrev ? onPrevMonthClick : undefined` instead